lib/rules/jsx-curly-brace-presence.js

```diff
@@ -59,18 +60,45 @@ module.exports = {
       {props: ruleOptions, children: ruleOptions} :
       Object.assign({}, DEFAULT_CONFIG, ruleOptions);
 
-    function containsBackslashForEscaping(rawStringValue) {
+    function containsLineTerminators(rawStringValue) {
+      return /[\n\r\u2028\u2029]/.test(rawStringValue);
+    }
+
+    function containsBackslash(rawStringValue) {
       return rawStringValue.includes('\\');
     }
 
+    function containsHTMLEntity(rawStringValue) {
+      return /&([A-Za-z\d#]+);/.test(rawStringValue);
```

Comment:
Final little nitpick: The parentheses in the regex are unnecessary. (Sorry, I can't help myself when I see a regex.)

Reply:
It's all good. I am actually quite grateful for all your regex comments. I learned a lot from them! 😄